pkg/pidfile/pidfile.go

```diff
@@ -33,6 +36,7 @@ func New(path string) (*PIDFile, error) {
 	if err := checkPIDFileAlreadyExists(path); err != nil {
 		return nil, err
 	}
+	system.MkdirAll(filepath.Dir(path), os.FileMode(0644))
```

Comment:
@aaronlehmann Updated. I didn't realise MkdirAll returns nil if a directory already exists, but verified this morning it does. So that makes error checking easier 😇 PTAL.